docs/DevelopmentGuide.md

```diff
@@ -80,6 +80,8 @@ the default Visual Studio style. See [clang-format style options](
 http://releases.llvm.org/3.6.0/tools/clang/docs/ClangFormatStyleOptions.html)
 for details.
 
+Note: For consistent formattting please use this version of [Clang/LLVM for Windows 64-bit](https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/LLVM-8.0.1-win64.exe).
```

Comment:
I think this should instead be stated in docs/GettingStarted.md line 13 (i.e., state the version).

But neither is wholly sufficient since the script says you can also run it from a bash shell, such as in WSL2, where a different clang version (not clang for windows) would be installed.  (Indeed I've run into cases I haven't tracked down yet where the script crashes when run in powershell but the bash version works.)

And if different versions have different formats, then we should update the format-code script to check the version, since no one will look here at this markdown by the time it will matter.

Reply:
Clang version 9 and below don't correctly format the code (it leaves lines > 80 characters long).
Clang version 10 and above does format code correclty.

Reply:
Got it, let's investigate switching to clang-10 then

Reply:
Change link to https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/LLVM-10.0.0-win64.exe ?